firefox/guide/firefox/firefox_preferences-ssl_protocol_tls/rule.yml

```diff
@@ -2,7 +2,8 @@ documentation_complete: true
 
 title: 'Enable TLS Usage in Firefox'
 
-description: 'To enable TLS, set <tt>security.enable_tls</tt> to <tt>true</tt>.'
+description: 'To enable TLS, set <tt>security.tls.version.min</tt> to <tt>2</tt>
+ and set <tt>security.tls.version.max</tt> to <tt>4</tt>.'
```

Comment:
This should really be a new rule since it is changing 2 separate configuration items.

Reply:
The following STIG requirement requires both values:

https://www.stigviewer.com/stig/mozilla_firefox/2019-12-12/finding/V-15983

We need to map a single rule to each requirement.

Reply:
> This should really be a new rule since it is changing 2 separate configuration items.

Understood. We need the rule to be broken out into 2 to meet the CCI.